README.md

```diff
@@ -85,6 +85,36 @@ res = client.delete(
 print(res)
 ```
 
+# migration tool
+Milvus Lite provides a command line tool that can dump milvus-lite data into a json file, which can be imported into [milvus](https://github.com/milvus-io/milvus) and [Zilliz Cloud](https://zilliz.com/cloud)(the fully managed cloud service for Milvus). The milvus-lite command  command will be installed together with milvus-lite python package 
+```shell
+milvus-lite dump -h
+
+usage: milvus-lite dump [-h] [-d DB_FILE] [-c COLLECTION] [-p PATH]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -d DB_FILE, --db-file DB_FILE
+                        milvus lite db file
+  -c COLLECTION, --collection COLLECTION
+                        collection that need to be dumped
+  -p PATH, --path PATH  dump file storage dir
+```
+export data:
```

Comment:
The following example dumps all data from xxx collection that's stored in xxx (Milvus Lite database file)